sdk/program/src/message.rs

```diff
@@ -8,10 +8,34 @@ use crate::{
     hash::Hash,
     instruction::{AccountMeta, CompiledInstruction, Instruction},
     pubkey::Pubkey,
-    short_vec, system_instruction,
+    short_vec, system_instruction, sysvar,
 };
 use itertools::Itertools;
-use std::convert::TryFrom;
+use lazy_static::lazy_static;
+use std::convert::{TryFrom, TryInto};
+
+lazy_static! {
+    // Copied keys over since direct references create cyclical dependency.
+    static ref BUILTIN_PROGRAMS_KEYS: [Pubkey; 12] = vec![
+        "Budget1111111111111111111111111111111111111",
+        "Config1111111111111111111111111111111111111",
+        "Exchange11111111111111111111111111111111111",
+        "ExchangeFaucet11111111111111111111111111111",
+        "FaiLure111111111111111111111111111111111111",
+        "NativeLoader1111111111111111111111111111111",
+        "Noop111111111111111111111111111111111111111",
+        "Stake11111111111111111111111111111111111111",
+        "StakeConfig11111111111111111111111111111111",
+        "Vest111111111111111111111111111111111111111",
+        "Vote111111111111111111111111111111111111111",
+        "ownab1e111111111111111111111111111111111111",
+    ]
```

Comment:
Should the SystemProgram be included here?

Reply:
added